airflow/runtimes/container_runtime.go

```diff
@@ -1,17 +1,59 @@
-package airflow
+package runtimes
 
 import (
 	"os"
 	"path/filepath"
 	"strings"
 	"sync"
+	"time"
+
+	"github.com/briandowns/spinner"
 
 	"github.com/astronomer/astro-cli/config"
 	"github.com/astronomer/astro-cli/pkg/fileutil"
 	"github.com/astronomer/astro-cli/pkg/util"
 	"github.com/pkg/errors"
 )
 
+var spinnerCharSet = spinner.CharSets[14]
+
+const (
+	docker                         = "docker"
+	podman                         = "podman"
+	containerRuntimeNotFoundErrMsg = "Failed to find a container runtime. " +
+		"See the Astro CLI prerequisites for more information. " +
+		"https://www.astronomer.io/docs/astro/cli/install-cli"
+	containerRuntimeInitMessage = " Astro uses container technology to run your Airflow project. " +
+		"Please wait while we get things started…"
+	spinnerRefresh = 100 * time.Millisecond
+)
+
+// ContainerRuntime interface defines the methods that manage
+// the container runtime lifecycle.
+type ContainerRuntime interface {
+	Initialize() error
+}
```

Comment:
This interface will grow to include more lifecycle functions when we follow up with podman support.